lib/matplotlib/tests/test_text.py

```diff
@@ -714,3 +714,9 @@ def test_update_mutate_input():
 def test_invalid_color():
     with pytest.raises(ValueError):
         plt.figtext(.5, .5, "foo", c="foobar")
+
+
+@image_comparison(['text_pdf_kerning.pdf'])
```

Comment:
Can you set new style, and regenerate the image? (Also squash the old image away.)
```suggestion
@image_comparison(['text_pdf_kerning.pdf'], style='mpl20')
```

Reply:
Sure, if migrated the code to the new style.